CHANGELOG.md

```diff
@@ -4,6 +4,166 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.1.0] - 2022-09-30
+
+## 🚀 Features
+
+### Build, test and publish binaries for `aarch64-unknown-linux-gnu` architecture ([Issue #1192](https://github.com/apollographql/router/issues/1192))
+
+We're now testing and building `aarch64-unknown-linux-gnu` binaries in our release pipeline and publishing those build artifacts as releases.  These will be installable in the same way as our [existing installation instructions](https://www.apollographql.com/docs/router/quickstart/).
+
+By [@EverlastingBugstopper](https://github.com/EverlastingBugstopper) in https://github.com/apollographql/router/pull/1907
+
+### Add ability to specify repository location in "DIY" Docker builds ([PR #1904](https://github.com/apollographql/router/issues/1904))
+
+The new `-r` flag allows a developer to specify the location of a repository when building a diy docker image. Handy for developers with local repositories.
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/1904
+
+### Support `serviceMonitor` in Helm chart
+
+`kube-prometheus-stack` ignores scrape annotations, so a `serviceMonitor` Custom Resource Definition (CRD) is required to scrape a given target to avoid `scrape_configs`.
+
+By [@hobbsh](https://github.com/hobbsh) in https://github.com/apollographql/router/pull/1853
+
+### Add support for dynamic header injection ([Issue #1755](https://github.com/apollographql/router/issues/1755))
+
+The following are now possible in our YAML configuration for `headers`:
+
+- Insert static header
+
+  ```yaml
+  headers:
+    all: # Header rules for all subgraphs
+      request:
+      - insert:
+          name: "sent-from-our-apollo-router"
+          value: "indeed"
+  ```
+
+- Insert header from context
+
+  ```yaml
+  headers:
+    all: # Header rules for all subgraphs
+      request:
+      - insert:
+          name: "sent-from-our-apollo-router-context"
+          from_context: "my_key_in_context"
+  ```
+
+- Insert header from request body
+
+  ```yaml
+  headers:
+    all: # Header rules for all subgraphs
+      request:
+      + insert:
```

Comment:
I believe this was a typo on https://github.com/apollographql/router/pull/1833 at https://github.com/apollographql/router/pull/1833/files#diff-de4ee5c7509f3bacb6d448d3cd29a8de401251038521352a41da4e09066952b3R61

```suggestion
      - insert:
```